arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/mmu/MmuCall.java

```diff
@@ -783,20 +782,30 @@ public static MmuCall forModexpPartialResultCopy(
   }
 
   public static MmuCall mcopyCopy(Hub hub, CallFrame callFrame) {
+    final EWord sourceOffset = EWord.of(callFrame.frame().getStackItem(1));
     final long size = clampedToLong(callFrame.frame().getStackItem(2));
     return new MmuCall(hub, MMU_INST_RAM_TO_RAM_SANS_PADDING)
         .sourceId(callFrame.contextNumber())
+        .sourceRamBytes(
+            Optional.of(
+                extractContiguousLimbsFromMemory(
+                    callFrame.frame(), Range.fromOffsetAndSize(sourceOffset.toLong(), size))))
         .targetId(newIdentifierFromStamp(hub.stamp()))
-        .sourceOffset(EWord.of(callFrame.frame().getStackItem(1)))
+        .sourceOffset(sourceOffset)
         .size(size)
         .referenceSize(size);
```

Comment:
Sanity check: the referenceOffset is implicitly set to 0 (given that it's not set by hand), right ?